openapiv2/artifact/service.swagger.yaml

```diff
@@ -16,12 +28,15 @@ paths:
           description: A successful response.
           schema:
             $ref: '#/definitions/v1alphaListKnowledgeBasesResponse'
+        "401":
+          description: Returned when the client credentials are not valid.
+          schema: {}
         default:
           description: An unexpected error response.
           schema:
             $ref: '#/definitions/rpcStatus'
       parameters:
-        - name: ownerId
+        - name: namespaceId
```

Comment:
This will require extra maintenance (I think this can be achieved by adding `field_configuration: {path_param_name: ...}` in the proto contract) but `namespaceID` (and similar initialisms) look so much better.

Reply:
Agree that initialisms `namespaceID` looks better. The issue with initialisms isn't just in the path parameters, it also appears in query parameters and even in the body. If we want to change this, we should do it all at once for consistency.

Reply:
Yeah I hadn't thought of that. Perhaps keeping the configuration in all the fields is a bit verbose and a maintenance burden. Drawbacks of using `camelCase` in a protocol opinionated about  `snake_case` 🙃